### ISSUE TITLE
[Equalizer] Make it work inside elements not displayed yet

### ISSUE THREAD

**Issue by cesasol** [zurb/foundation-sites#9359]
I guess this same logic can be applied for all plugins that does not work when the initial state is display:none.

**How can we reproduce this bug?**

1. Make tabs
2. Inside a not initially active tab put an equalizer row
3. The equalizer does not know the height since the parent is display:none and has 0 height 

**What did you expect to happen?**

Seeing equalizer and other plugins that depend on current values work out of the box or with a simple extra 'data' element

**What happened instead?**

The equalizer gives 0 height to all elements watched

**Test case:**

- [Example](http://codepen.io/cesasol/pen/jVWwxb)

**Comment by cesasol** [comment]
I have made a fix that can be a solution for anyone having the same problem
```` js
var panelesMenu = {};
function cargarDormamu() {
  $('#menu-tabs-content .tabs-panel').each(function(index) {
    console.log(index);
    panelesMenu[$(this).attr('id')] =  new Foundation.Equalizer($(this), {
        equalizeByRow: false,
        equalizeOn: 'medium'
      });
    });
  console.log(panelesMenu);
}
$('#menu-tabs').on('change.zf.tabs', function (event, object) {
  console.log('cambiado');
  var a = $(object).find('a').prop("hash"),
  b = a.substring(1)
  ajustarEqu(panelesMenu[b])
});

function ajustarEqu(elem) {
  console.log(elem);
  if (elem) {
    elem.getHeights(function(esto) {
      elem.applyHeight(esto);
      console.log(esto);
    });
  }
}
````

**Comment by coreysyms** [comment]
This has been fixed for 6.3 RE: #9320
